src/structures/MessageMentions.js

```diff
@@ -21,6 +21,9 @@ class MessageMentions {
       } else {
         this.users = new Collection();
         for (const mention of users) {
+          if (mention.member && message.guild) {
+            message.guild.members.set(mention.id, Object.assign(mention.member, { user: mention }));
```

Comment:
Basically the same as above.

Reply:
Resolved by https://github.com/discordjs/discord.js/pull/3684/commits/ffd61ebfa8e8034db18b79ea66bf0d2265e4975c (I hope)